suite-native/graph/src/components/GraphFiatBalance.tsx

```diff
@@ -34,7 +34,7 @@ const Skeleton = () => (
 
 const Balance = ({ selectedPointAtom }: BalanceProps) => {
     const point = useAtomValue(selectedPointAtom);
-    const fiatValue = String(point.value);
+    const fiatValue = point.valueLatestTotal ? point.valueLatestTotal : String(point.value);
```

Comment:
nitpick:
same, but easier to read?
```suggestion
    const fiatValue = point.valueLatestTotal || String(point.value);
```